.github/workflows/code-inspector.yaml

```
@@ -1,42 +1,91 @@
+      - name: Install dependencies
+        run: pnpm install
+      - name: Test
+        env:
+          TZ: Asia/Shanghai
+          NODE_OPTIONS: --import tsx
+        run: pnpm test -- --reporter=default --reporter=github-actions
+
+  test-coverage:
+    name: Test Coverage
+    # Temporarily disabled; enable by removing this guard when coverage is ready.
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)